README.md

```diff
@@ -1,2 +1,359 @@
+the UI; its value can be either enabled, disabled, or unspecified - in which
+case, it will default to the configuration value defined in the [port-security-enabled-default](https://github.com/oVirt/ovirt-provider-ovn#section-network)
+attribute.
+
+#### Updating port security on ports or networks
+
+In this section it is shown how to use the sample ansible playbook to configure
+the port security of networks and ports can be found.
+
+```bash
+# update the port security of a single port
+ansible-playbook -i localhost update_port_security.yml \
+  --extra-vars="port_security_enabled=true port_uuid=<the_port_uuid>"
+
+# update the port security of a all ports attached to a logical network
+# this playbook also updates the port security of all ports connected
+# to the given network
+ansible-playbook -i localhost update_port_security.yml \
+  --extra-vars="port_security_enabled=true network_uuid=<the_network_uuid>"
+
+# update the port security of all logical ports found in the system
+ansible-playbook -i localhost update_port_security.yml \
+  --extra-vars="port_security_enabled=true"
+
+```
+
+### Entity objects on the networking API
+The two relevant entities on the networking API are **security groups** and
+**security group rules**.
+
+A security group operates as a container of rules, and can be applied to ports.
+The intended use case, is for a user to create a group, add a set of rules to
+it, and finally, attach the group to a (set of) port(s).
+
+A security group rule specifies which type of traffic (using L3 / L4 semantics)
+can access - or exit - the VM. They are scoped to the security group - e.g. the
+rules can have the same data **whenever** they belong to different groups.
+A security group rule can also be applied to either *ingress* - traffic incomming to the VM - or *egress* - traffic outgoing from the VM.
+
+The *knobs* on a security group are:
+* IP protocol
+* ethertype - ipv4 / ipv6
+* L4 port range - use max == min to open a single port
+* remote ip prefix (using a CIDR), or a single IP
+* remote group membership
+* direction
+
+Their APIs can be found in:
+* [security groups](https://developer.openstack.org/api-ref/network/v2/#security-groups-security-groups)
+* [security group rules](https://developer.openstack.org/api-ref/network/v2/#security-group-rules-security-group-rules)
+
+### Default group
+The **Default** group's goal is to allow access between all VMs out of the box.
+
+To implement the Default group, four rules are used:
+* allow all ingress Ipv4 belonging to the Default group
+* allow all ingress Ipv6 belonging to the Default group
+* allow all egress Ipv4
+* allow all egress Ipv6
+
+When a VM is created, having **port-security-enabled** active, will be
+automatically added to the **Default** group, which will allow all VMs
+to communicate by default - e.g. the **ingress** rules will be matched,
+and traffic allowed into the VM.
+
+The outgoing traffic - e.g. will also match the egress rules - thus allowing
```

Comment:
i think e.g. is redundant here.

Reply:
Rephrased it. You're right, whoever wrote that sounds pretentious.  And ignorant. 0_o